__tests__/index.test.js

```diff
@@ -153,3 +162,128 @@ describe('GET api/articles', () => {
+        .then(({body}) => {
+            const {comments} = body;
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                })
+            })
+        })
+    })
+    test('GET: 200 repsonds with an array of all comments for an article_id specified on the endpoint', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body;
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                })
+                expect(comment.article_id).toBe(1);
+            })
+        })
+    })
+    test('GET: 200 repsonds with an array of all comments for an article_id specified on the endpoint, returned in order with most recent first', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body;
+            expect(comments).toBeSortedBy('created_at', {descending: true});
+        })
+        
+    })
+    test('GET: 404 repsonds with an error message resource not found, when invalid id request made on the endpoint', () => {
+        return request(app)
+        .get('/api/articles/100/comments')
+        .expect(404)
+        .then(({body}) => {
+            const {msg} = body;
+            expect(msg).toBe('Resource not found');
+        })
+    })
+    test('GET: 400 repsonds with an error message bad request, when bad request made on the endpoint', () => {
+        return request(app)
+        .get('/api/articles/hello/comments')
+        .expect(400)
+        .then(({body}) => {
+            const {msg} = body;
+            expect(msg).toBe('Bad request');
+        })
+    })
+})
+
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('POST: 201 responds with a new comment added for an article with article_id specified in endpoint', () => {
+        const newComment = {
+            username: 'icellusedkars',
+            body: 'This is a new comment',
+          };
+        return request(app)
+        .post('/api/articles/3/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({body}) => {
+            const {comment} = body;
+            expect(comment).toMatchObject(
+                {
```

Comment:
we would also expect comment_id, votes, and created at

Reply:
inside the object you can use {comment_id: expect.any(Number)} when you know the type but not the exact value, same for String ect, look at jest docs for more information